spec.html

```diff
@@ -8252,7 +8252,7 @@ <h1>MakeArgGetter ( _name_, _env_ )</h1>
           <p>The abstract operation MakeArgGetter called with String _name_ and Environment Record _env_ creates a built-in function object that when executed returns the value bound for _name_ in _env_. It performs the following steps:</p>
           <emu-alg>
             1. Let _steps_ be the steps of an ArgGetter function as specified below.
-            1. Let _getter_ be CreateBuiltinFunction(_steps_, &laquo; [[Name]], [[Env]] &raquo;).
+            1. Let _getter_ be ! CreateBuiltinFunction(_steps_, &laquo; [[Name]], [[Env]] &raquo;).
```

Comment:
GetBindingValue can throw a ReferenceError
```suggestion
            1. Let _getter_ be ? CreateBuiltinFunction(_steps_, &laquo; [[Name]], [[Env]] &raquo;).
```

Reply:
I think that you suggestion is valid. I can't commit your change it while conversation is marked as resolved though

Reply:
For some reason I can't commit it even when conversation is marked as unresolved. It says that diff has been updated, but it's actually not  ¯\_(ツ)_/¯ 
<img width="410" alt="image" src="https://user-images.githubusercontent.com/1507086/60525077-6a405680-9cf6-11e9-9382-87d29d34a3f4.png">